driver/razerchromacommon.c

```diff
@@ -1193,6 +1193,36 @@ struct razer_report razer_chroma_misc_set_orochi2011_poll_dpi(unsigned short pol
     return report;
 }
 
+/**
+ * Set the Naga Trinity to "Static" effect
+ *
+ * Status Trans Packet Proto DataSize Class CMD Args
+ * 00     1f    0000   00    0e       0f    03  000000000200ffaa00ffaa00ffaa | SET LED MATRIX Effect (NOSTR, Main light (0x00), Static 0x00 (Unknown), Static 0x00 (Unknown), RGB 0x00FFAA, RGB 0x00FFAA, RGB 0x00FFAA)
+ * 00     1f    0000   00    0e       0f    03  0000000002aa00ffaa00ffaa00ff | SET LED MATRIX Effect (NOSTR, Main light (0x00), Static 0x00 (Unknown), Static 0x00 (Unknown), RGB 0xAA00FF, RGB 0xAA00FF, RGB 0xAA00FF)
```

Comment:
I'm assuming this comment isn't correct so remove it please

Reply:
Done